modules/node/handler/nodes_handler.e

```diff
@@ -49,10 +52,68 @@ feature -- HTTP Methods
 			create {GENERIC_VIEW_CMS_RESPONSE} l_page.make (req, res, api)
 			l_page.add_variable (node_api.nodes, "nodes")
 
+			create pager.make (api, node_api)
+			number_of_pages := (node_api.nodes_count // pager.limit) + 1
+
+				-- Size:limit
+			if
+				attached {WSF_STRING} req.query_parameter ("size") as l_size and then
+				l_size.is_integer
+			then
+				pager.set_limit (l_size.integer_value.to_natural_32)
+			end
+
+
+
+				--Page:offset
+			if
+				attached {WSF_STRING} req.query_parameter ("page") as ll_page and then
+				ll_page.is_integer
+			then
+				current_page := ll_page.integer_value
+				if current_page > 1 then
+					pager.set_offset (((current_page-1)*(pager.limit.to_integer_32)).to_natural_32)
+				end
+			else
+				current_page := 1
+			end
+
+
 
 				-- NOTE: for development purposes we have the following hardcode output.
 			create s.make_from_string ("<p>Nodes:</p>")
```

Comment:
Maybe we should use "Items" rather than "Nodes" ... and use the translation function for future i18n changes.